test/EFCore.PG.FunctionalTests/EFCore.PG.FunctionalTests.csproj

```diff
@@ -26,6 +26,6 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="$(EFCoreVersion)" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Relational.Specification.Tests" Version="$(EFCoreVersion)" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="$(MicrosoftExtensionsVersion)" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="$(MicrosoftExtensionsVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="2.2.4" />
```

Comment:
Why not `$(MicrosoftExtensionsVersion)`?

Reply:
Just that this one had a higher patch available, but the other extensions didn't. We could just keep them all pegged instead?

Reply:
Oh, I see. In that case that's fine - we shouldn't definitely use the latest one possible. This is why prematurely extracting to common variables doesn't always work :)